src/Dotnet.Script.DependencyModel/ProjectSystem/csproj.template

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>  
+  </PropertyGroup>
+  <ItemGroup>    
```

Comment:
this empty group can be removed

Reply:
The `<ItemGroup>`  is a placeholder for package references. Could have been added from code, but I see no actual harm having it in the template.